### ISSUE TITLE
Limit to Bank Money

### ISSUE THREAD

**Issue by chrismclaughlin55** [usc-csci201-fall2013/simcity201#23]
Do the banks have endless amounts of money, or are they limited by how much money has been deposited into the accounts? Similar to the cook/market scenario where if the market ran out of something then there was no way for the cook to order more of that food. Is this something we should worry about in our bank system?
The issue I had in mind was when a customer wants to withdraw money but the bank has loaned out all or most of its money to a point where it doesn't have enough money to cover the withdraw. 

@alghanmi @mahdiazmandian @dwilczyn

**Comment by dwilczyn** [comment]
Yes, banks have limited amounts of money, but for now let's assume the banks can cover withdrawals. We can consider a "run" on the bank (more withdrawals than the bank has money for) in our non-norms.

**Comment by gabelew** [comment]
Could we implement it such that the bank has a limited but separate supply of money for loans, and withdrawals only happen within the context of the customer's account? Then customers won't run into the problem of withdrawing money they don't have or failing to withdraw their own money that has been taken by somebody else.

**Comment by dwilczyn** [comment]
I will accept anything reasonable.
